src/main/java/com/mixpanel/android/mpmetrics/MixpanelAPI.java

```diff
@@ -1499,6 +1518,14 @@ public void trackNotification(final String eventName, final InAppNotification no
         }
 
         @Override
+        public void joinExperimentIfAvailable() {
+            final JSONArray variants = mDecideMessages.getVariants();
```

Comment:
I'm wondering if we want to call it `variants` instead of `changes`. We use `variants` for describing different A/B test versions on web UI.